database/src/main/resources/db/migration/V8_0__add_tx_request_view.sql

```diff
@@ -0,0 +1,90 @@
+UPDATE coin_mint SET status = 'QUEUED' WHERE status = 'INSERTED';
```

Comment:
@ranadewall do we have a good environment where we can test this migration against? not that most of these matter too much given we're no longer using some of these tables

Reply:
all the statuses in our internal test env are completed.

Reply:
sounds pretty low risk given those in prod should also be all completed

Reply:
especially if we could get them to upgrade asap - the usdf and nycb dcc coin balance is 0 this morning.